app/src/main/java/org/ole/planet/myplanet/ui/team/TeamPagerAdapter.kt

```diff
@@ -26,22 +26,16 @@ class TeamPagerAdapter(fm: FragmentActivity, team: RealmMyTeam?, private val isI
     private val isEnterprise: Boolean = TextUtils.equals(team?.type, "enterprise")
 
     init {
+        list.add(MainApplication.context.getString(R.string.chat))
         list.add(MainApplication.context.getString(if (isEnterprise) R.string.mission else R.string.plan))
         list.add(MainApplication.context.getString(if (isEnterprise) R.string.team else R.string.joined_members))
         if (isInMyTeam || team?.isPublic == true) {
-            list.add(MainApplication.context.getString(R.string.chat))
+            if (isEnterprise) list.add("Reports")
```

Comment:
moving the position of this changes the position of the reports tab in the ui. maintain original position
`current position`
![Screenshot_20240711_131716](https://github.com/open-learning-exchange/myplanet/assets/64019708/439974b4-4261-44a2-8293-ccc9671810a6)

`original position` how it should be

![Screenshot_20240711_131858](https://github.com/open-learning-exchange/myplanet/assets/64019708/4caa1fdc-4c20-4bb3-8f78-5bd2c6a36306)